kratos/processes/variational_distance_calculation_process.h

```diff
@@ -457,6 +457,20 @@ class VariationalDistanceCalculationProcess : public Process
     ///@name Input and output
     ///@{
 
+    static Parameters GetSpecifications(const Parameters rParameters)
+    {
+        const Parameters specifications = Parameters(R"({
+            "required_solution_step_data_variables" : ["DISTANCE"],
```

Comment:
we shall suggest also a list of non-historical variables.

"required_nonhistorical_nodal_variables" for example

i would also suggest to 

"required_solution_step_data_variables" -> "required_historical_nodal_variables"

also add list of elemental and condition variables